client/src/components/reviews/reviewlist.jsx

```diff
@@ -26,9 +35,11 @@ const ReviewList = ({ productID, mainData }) => {
 
   const reviewWindow = document.getElementById('reviewAllTiles');
   const scrollHandler = () => {
-    if (reviewWindow.scrollTop + reviewWindow.offsetHeight + 2 >= reviewWindow.scrollHeight && term === '') {
-      setCount(count + 2);
-      setScrolledCount(count + 2);
+    if (clicked) {
+      if (reviewWindow.scrollTop + reviewWindow.offsetHeight + 2 >= reviewWindow.scrollHeight && term === '') {
+        setCount(count + 2);
+        setScrolledCount(count + 2);
```

Comment:
what does this scrollHandler do exactly? looks interesting.

Reply:
makes it so the count increases when bottom of the scroll window is reached. Increasing the count increases number of review tiles rendered.